x-pack/plugins/apm/server/routes/ui_filters.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Boom from 'boom';
+import Joi from 'joi';
+import { CoreSetup } from 'src/core/server';
+import { withDefaultValidators } from '../lib/helpers/input_validation';
+import { setupRequest } from '../lib/helpers/setup_request';
+import { getEnvironments } from '../lib/ui_filters/get_environments';
+
+const defaultErrorHandler = (err: Error) => {
+  // eslint-disable-next-line
+  console.error(err.stack);
+  throw Boom.boomify(err, { statusCode: 400 });
```

Comment:
I realize this is just moving code around, but should the default status not be 500 (our fault, not yours) instead of 400 (your fault, not ours)?

Reply:
Yeah. you might have a point. I think we should change all of them in a separate PR though.